protocols/indexer/reports/analysis/slither.txt

```diff
@@ -1,5 +1,5 @@
 INFO:Detectors:
-Indexer.unsetIntent(address,address) (contracts/Indexer.sol#168-195) ignores return value by external calls "indexes[_makerToken][_takerToken].unsetLocator(msg.sender)" (contracts/Indexer.sol#186)
+Indexer.unsetIntent(address,address) (contracts/Indexer.sol#168-195) ignores return value by external calls "indexes[_signerToken][_senderToken].unsetLocator(msg.sender)" (contracts/Indexer.sol#186)
```

Comment:
would not change these as they're autogenerated, likely we should just remove this file.